source/declarations.tex

```diff
@@ -2563,13 +2565,23 @@
 
 \pnum
 In a declaration
-\opt{\grammarterm{attribute-specifier-seq}}
 \tcode{T}
 \tcode{D}
 where
 \tcode{D}
-is an unadorned identifier the type of this identifier is
-``\tcode{T}''.
+has the form
+\begin{ncsimplebnf}
+declarator-id \opt{attribute-specifier-seq}
+\end{ncsimplebnf}
+the type of the
+\grammarterm{declarator-id}
+in
+\tcode{D}
```

Comment:
I think  "in D" can go here; there is only one declarator-id we talk about in this context.

Reply:
Ah, no, that's the pattern we use elsewhere, too.